sycl/test/basic_tests/SYCL-2020-spec-constants.cpp

```diff
@@ -38,6 +38,8 @@ constexpr sycl::specialization_id<uint64_t> uint64_id(8);
 constexpr sycl::specialization_id<float> float_id(10.0);
 constexpr sycl::specialization_id<double> double_id(11.0);
 constexpr sycl::marray<double, 5> ma;
+constexpr sycl::marray<double, 5> mb(ma);
```

Comment:
It is not clear why these lines were added to the test?
tey are not used in specialization constants and this test is for specialization constants.

Reply:
Agree, I've moved the lines into `marray.cpp` test.